requirements.txt

```diff
@@ -23,12 +23,12 @@ docker==4.4.3
 dvc[ssh]==1.11.16
 gunicorn==20.0.4
 h5py==3.1.0
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.1/en_core_web_sm-2.3.1.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.5/en_core_sci_lg-0.2.5.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.5/en_ner_bc5cdr_md-0.2.5.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.5/en_ner_bionlp13cg_md-0.2.5.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.5/en_ner_craft_md-0.2.5.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.5/en_ner_jnlpba_md-0.2.5.tar.gz
+https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_lg-0.4.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bc5cdr_md-0.4.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bionlp13cg_md-0.4.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_craft_md-0.4.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_jnlpba_md-0.4.0.tar.gz
```

Comment:
Sorry for probably a stupid questions, but are the new models the same as the old ones in terms of machine learning? In other words, are they making exactly the same predictions (=same weights + same architecture)?

In your instructions you described changes of NER performance, is this because the new models are different than the old ones?

Reply:
No. No.

It seems so.

I have added in the PR description a performance comparison for the models between their version 0.2.5 and 0.4.0.
The performance drops in our models seem to come from a performance drop in the associated base models.

Reply:
So is it possible to load the old models with new `spacy`? Or are there some breaking changes that make it impossible?

Reply:
No. Yes.

Since my last comment, I have continued the investigations on the performance changes.

The performance drops are not completely explained by the performance changes in the base models.

Reply:
UPDATE on the performance changes in https://github.com/BlueBrain/Search/pull/248#issuecomment-810389551.